scripts/code_coverage_report/generate_code_coverage_report/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
```

Comment:
Use a section in the root of the repo for any .gitignore entries.

Reply:
> How can I see the generated reports?

A report can be seen in the pr. A tested [report](https://github.com/firebase/firebase-ios-sdk/pull/7357#issuecomment-768579132) for database is in this pr. We will only have database and functions at this moment. After testing the coverage storage on Metrics Service, we can involved more SDKs.